MapboxNavigation/CarPlayManagerDelegate.swift

```diff
@@ -165,6 +165,69 @@ public protocol CarPlayManagerDelegate {
      - parameter carPlayManager: The CarPlay manager instance.
      - returns: A Boolean value indicating whether to disable idle timer when carplay is connected and enable when disconnected.
      */
-    @objc optional func carplayManagerShouldDisableIdleTimer(_ carPlayManager: CarPlayManager) -> Bool
+    func carplayManagerShouldDisableIdleTimer(_ carPlayManager: CarPlayManager) -> Bool
+}
+
+@available(iOS 12.0, *)
+extension CarPlayManagerDelegate {
```

Comment:
This default implementation extension doesn’t work: the CarPlay example application fails to build with the following error:

```
/path/to/mapbox-navigation-ios/Example/AppDelegate+CarPlay.swift:47:1: Type 'AppDelegate' does not conform to protocol 'CarPlayManagerDelegate'
/path/to/mapbox-navigation-ios/Example/AppDelegate+CarPlay.swift:47:1: Do you want to add protocol stubs?
```

And the fix-it adds protocol stubs for all the methods implemented below. I think the problem is that the extension is internal whereas the protocol is public.